tensorflow/compiler/tf2tensorrt/convert/ops/softmax.cc

```diff
@@ -0,0 +1,67 @@
+#if GOOGLE_CUDA && GOOGLE_TENSORRT
```

Comment:
Please add license header, can copy and paste from another source file then change the year to 2022.